installer/charts/rhtap-subscriptions/templates/delete_tekton_config.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: delete-tekton-config-if-exists
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    "helm.sh/hook": post-install
+    "helm.sh/hook-delete-policy": hook-succeeded
+spec:
```

Comment:
There's a couple of issues raised by SonarCloud.

Reply:
I saw that, will take care of those

Reply:
done